src/components/GameView/GameOverlays.vue

```diff
@@ -18,60 +19,88 @@
         Leave Game
       </v-btn>
     </v-overlay>
+
     <v-overlay
       id="waiting-for-opponent-counter-scrim"
       v-model="waitingForOpponentToCounter"
-      class="game-overlay"
+      class="d-flex flex-column justify-center align-center"
     >
-      <h1 :class="[this.$vuetify.display.xs === true ? 'text-h5' : 'text-h3']">
+      <h1 :class="[this.$vuetify.display.xs === true ? 'text-h5' : 'text-h3', 'overlay-header']">
         {{ showWaitingForOpponetToCounterMessage }}
       </h1>
+      <div id="counter-scrim-cards">
+        <game-card
+          v-if="oneOff"
+          :rank="oneOff.rank"
+          :suit="oneOff.suit"
+          :data-overlay-one-off="`${oneOff.rank}-${oneOff.suit}`"
+          :jacks="twos"
+          class="overlay-card"
+        />
+        <div>
+          <game-card
+            v-for="(two, index) in twos"
+            :key="`overlay-two-${two.id}`"
+            :rank="two.rank"
+            :suit="two.suit"
+            :data-overlay-counter="`${two.rank}-${two.suit}`"
+            :class="`overlay-card overlay-two overlay-two-${index}`"
```

Comment:
These classes are used to apply the different rotation transforms to each two in this list